Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1589,6 +1589,11 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
         controller.addAction(UIAlertAction(title: Strings.NewPrivateTabTitle, style: .default, handler: { _ in
             self.tabManager.addTabAndSelect(isPrivate: true)
         }))
+        controller.addAction(UIAlertAction(title: Strings.CloseTabTitle, style: .default, handler: { _ in
```

Comment:
I think the style for this should be .destructive because this is a delete action

Reply:
:+1: 

I ummed and ahhed about using `.destructive`. 

Reasons against: "Close All Tabs" isn't `.destructive`; now adding a third color to our menus: black, blue and red.